<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Dialog</class>
 <widget class="QDialog" name="Dialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>759</width>
    <height>366</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Dialog</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="1" column="0" rowspan="2" colspan="4">
    <widget class="QListView" name="list"/>
   </item>
   <item row="3" column="0">
    <widget class="QCheckBox" name="checkBox">
     <property name="text">
      <string>Preview</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>Recherche</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1" colspan="3">
    <widget class="QLineEdit" name="ln_search"/>
   </item>
   <item row="1" column="4" rowspan="2">
    <widget class="AssetViewer" name="fileViewer" native="true"/>
   </item>
   <item row="3" column="1" colspan="3">
    <widget class="QDialogButtonBox" name="buttonBox_2">
     <property name="standardButtons">
      <set>QDialogButtonBox::Cancel|QDialogButtonBox::Ok</set>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>AssetViewer</class>
   <extends>QWidget</extends>
   <header>FileManager/FileViewer.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>accepted()</signal>
   <receiver>Dialog</receiver>
   <slot>accept()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>404</x>
     <y>341</y>
    </hint>
    <hint type="destinationlabel">
     <x>539</x>
     <y>280</y>
    </hint>
   </hints>
  </connection>
  <connection>
   <sender>buttonBox_2</sender>
   <signal>rejected()</signal>
   <receiver>Dialog</receiver>
   <slot>reject()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>425</x>
     <y>349</y>
    </hint>
    <hint type="destinationlabel">
     <x>613</x>
     <y>273</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>search(QString)</slot>
 </slots>
</ui>
